esis/science/papers/instrument/__init__.py

```diff
@@ -1180,49 +1167,47 @@ def document() -> kgpy.latex.Document:
-aperture of the telescope) and the spatial extent of the \CCDs.
-The \FOV\ in the $m=\pm1$ orders is shifted along the dispersion axis as a function of wavelength, dependant upon where 
-the dispersed spectral images intercept the $m=\pm1$ \CCDs.
-Spatially, this effect is limited to only a handful of pixel columns at the edges of each image order.
-Of higher concern is the ``spectral contamination'' allowed by this layout; 
-\citet{Parker16} found that bright spectral lines and continuum far outside the wavelength passband and nominal $m=0$ 
-\FOV\ could be diffracted onto the outboard order \CCDs.
-This off-band contamination is detected as systematic intensity variation that lacks an anti-symmetric pairing in the 
-opposite dispersed image order.
-\sout{Analysis of the spectral contamination is ongoing.}
-\roy{Is there a way we can update this?}
-
-Finally, the exposure cadence of \MOSES\ is hindered by an $\sim$\SI{6}{\second} readout time for the 
-\CCDs~\citep{Fox11}.
-The observing interval for a solar sounding rocket flight is very short, typically about five minutes.
-Consequently, every second of observing time is precious, both to achieve adequate exposure time and to catch the full 
-development of dynamical phenomena.
-The \MOSES\ observing duty cycle is $\sim$\SI{50}{\percent} since it is limited by the readout time of its \CCDs.
-Thus, valuable observing time is lost.
-The readout data gap \sout{impelled} \roy{compelled} us to develop a \MOSES\ exposure sequence with exposures ranging from 
-$0.25$-\SI{24}{\second}, a careful trade-off between deep and fast exposures. 
+The single diffraction grating also leads to a compromise in the optical performance of the instrument. Since the 
+\MOSES\ grating forms images in three orders simultaneously, \sout{aberration cannot be simultaneously optimized for} 
+\roy{there aren't enough degrees of freedom in the optical system to achieve sub-pixel aberrations in} all three of 
+those spectral orders. A result of this design is that the orientations (\ie\,the major axis) of the \PSF\ varies 
+order to order~\citep{Rust17}. During the first mission, \MOSES\ was flown with a small amount of defocus~\citep{
+Rust17}, which exacerbated the inter-order \PSF\ variation and caused the individual \PSFs\ to span several 
+pixels~\citep{Rust17,Atwood18}. The combination of these two effects results in spurious spectral features that 
+require additional consideration~\citep{Atwood18} and further increase the complexity of the inversion 
+process~\citep{Rust17,Courrier18}. 
+
+\jake{Another complication with \MOSES\ is that each spectral order image contains a different combination of spatial 
+and spectral information.  This stems from the fact that \MOSES\ lacks a field stop to define a wavelength 
+independent \FOV and that it uses an undispersed channel. In this configuration, intensity from wavelengths off the 
```

Comment:
Sentence about how MOSES's zero order is important for it's spatial resolution parallel to dispersion.